meta-security-smack/recipes-core/systemd/systemd/udev-smack-default.rules

```diff
@@ -0,0 +1,23 @@
+# do not edit this file, it will be overwritten on update
```

Comment:
Where did this file come from? If you wrote it, are you sure that you have captured all the cases where Smack labels need to be set?